@xen-orchestra/rest-api/src/alarms/alarm.service.mts

```diff
@@ -40,14 +40,15 @@ export class AlarmService {
     return {
       ...alarm,
       body: {
-        value, // Keep the value as a string because NaN, Infinity, -Infinity is not valid JSON
+        value: Number.isFinite(+value) ? (+value * 100).toFixed(1) : value, // Keep the value as a string because NaN, Infinity, -Infinity is not valid JSON
```

Comment:
Would you like to check if it is below 1 to be sure? I suppose it can't be upper than 1 but it would be my first reflex to test:
```suggestion
        value: Number.isFinite(+value) && +value <= 1 ? (+value * 100).toFixed(1) : value, // Keep the value as a string because NaN, Infinity, -Infinity is not valid JSON
```

Reply:
Some alarms may have a value greater than 1.
E.g.: `mem_usage: 1.05`